sdks/python/apache_beam/io/fileio.py

```diff
@@ -796,17 +796,24 @@ def __iter__(self):
         # that the end offset of the range may be smaller than the original
```

Comment:
might be nice to stay stylistically consistent unless there is some reason I'm missing.

```python
try:
   ...
finally:
  ...
```
then break vs return is not a big deal.